src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -1735,6 +1732,26 @@ def test_network_subnet_endpoint_service(self, resource_group):
                  checks=self.check('serviceEndpoints', None))
 
 
+    #@ResourceGroupPreparer(name_prefix='cli_subnet_delegation')
```

Comment:
revert comment unless you plan on remembering when merging to dev

Reply:
Ah, thanks. I did this to test the checks but yes, I want to revert it now.